app/_locales/en/messages.json

```diff
@@ -1126,7 +1126,7 @@
     "message": "Gas limit is the maximum amount of units of gas you are willing to spend."
   },
   "gasLimitRecommended": {
-    "message": "Recommended gas limit is $1. If the gas limit is less than estimated transaction can fail."
+    "message": "Recommended gas limit is $1. If the gas limit is less that, it may fail."
```

Comment:
@danjm You can see changes here realted to: https://github.com/MetaMask/metamask-extension/pull/12784#discussion_r761033281